### PR TITLE
Add basic config

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- There is no way to login. #1 
- There is no connection to our Firebase service. #2 
- There is no dashboard component. #3 
- There are no maps for project I.N.S.I.G.H.T. #4 
- There is no map for active HYDRA agents. #4 

**What is the new behavior (if this is a feature change)?**
- A login has been added. #1 
- The connection to the Firebase service has been established. #2 
- A dashboard component has been added. #3 
- Two maps have been added to display project I.N.S.I.G.H.T. targets and active HYDRA agents. #4 

**Additional informations**
- The version has been updated to 0.1.1

### CODE REVIEW

**Review by Aichnerc — COMMENTED** [review]
It should be okay to merge, after some max. char-length per line changes.

File: src/components/pages/SignIn/index.jsx [hunk 1]
```diff
@@ -61,48 +71,114 @@ class SignIn extends React.Component {
...
+                                                        Please enter your password.
+                                                    </div>
+                                                </div>
+                                            </MDBCol>
+                                        </MDBRow>
+
+                                        {  }
                                         <div className="text-center mt-4">
-                                            <MDBBtn color="success" type="submit"><i className="fas fa-key pr-2"></i>Login</MDBBtn>
+                                            <MDBBtn color="danger" type="submit"><i className="fas fa-key pr-2"></i>Login</MDBBtn>
```
Comment (sweggerat) [inline src/components/pages/SignIn/index.jsx]: Max characters per line

File: src/components/pages/SignIn/index.jsx [hunk 2]
```diff
@@ -61,48 +71,114 @@ class SignIn extends React.Component {
...
                                         </div>
-                                        <p className="text-muted text-center mt-3">Not a member yet? No problem, just <Link to="/join"><strong>join us</strong></Link>!</p>
+                                        <p className="text-muted text-center mt-3">No access? <Link to="/join"><strong>Join</strong></Link> HYDRA now!</p>
```
Comment (sweggerat) [inline src/components/pages/SignIn/index.jsx]: Max characters per line

File: src/utilities/_fonts.scss [hunk 3]
```diff
@@ -0,0 +1,10 @@
+@import url('https://fonts.googleapis.com/css?family=Ubuntu&display=swap');
+
+.ubuntu {
+    font-family: 'Ubuntu', "Gill Sans", "Gill Sans MT", "Myriad Pro", "DejaVu Sans Condensed", Helvetica, Arial, sans-serif;
```
Comment (sweggerat) [inline src/utilities/_fonts.scss]: Max characters per line

**Review by sweggerat — CHANGES_REQUESTED** [review]
After this changes it should be ok to merge.

File: src/components/pages/SignIn/index.jsx (hunk 1 above)
Comment (Aichnerc) [reply src/components/pages/SignIn/index.jsx]: Fixed it!

File: src/components/pages/SignIn/index.jsx (hunk 2 above)
Comment (Aichnerc) [reply src/components/pages/SignIn/index.jsx]: Fixed it!

File: src/utilities/_fonts.scss (hunk 3 above)
Comment (Aichnerc) [reply src/utilities/_fonts.scss]: Fixed it!

**Review by sweggerat — APPROVED** [review]
Shoud be ok to merge

**Pull request merged**